templates/environment/versions/cf-v1.4.0.yml

```diff
@@ -366,7 +366,7 @@ Outputs:
     Export:
       Name: !Sub ${AWS::StackName}-SubDomain
   EnabledFeatures:
-    Value: !Sub '${ALBWorkloads},${EFSWorkloads},${NATWorkloads}'
+    Value: !Sub '${ALBWorkloads},${EFSWorkloads},${NATWorkloads},${Aliases}'
```

Comment:
I wonder if we can hit the limit of 4096 bytes here for this output if a user has enough features enabled now 💭 : https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cloudformation-limits.html

I think `Aliases` is actually safe and we don't have to include it in this list because it's referred here:
https://github.com/aws/copilot-cli/blob/22e3b8d52ec03d98f8ce6ca24ba503618c636db5/templates/environment/partials/custom-resources.yml#L61

So if it's value is changed it will always trigger a stack update so we should be safe to remove it from here

Reply:
Maybe we can just add a comment saying we don't need to include Aliases because updating it always results in the CustomDomain action to update